### DISCUSSION TITLE
Developer Mode( How to enable) and/or How to activate Linux beta

### DISCUSSION THREAD

**Discussion by dedvoltg** [sebanc/brunch#1278] [Q&A]
I have a windows hp laptop that I re imaged to chrome OS grunt 93 with the latest brunch as of October 28th 2021. I really need to activate developer mode but I don't know how to on this computer. I have already tried the hold esc + refresh/f3 + power way doesn't work. My main goal of doing this is to get the Linux beta option on my laptop. Nether in the developer tab in advance settings section is present nor the Linux beta tab in not present in my settings. If someone was able to fix one of these problems (preferably the Linux one first) of my problems I would very happy. 

Thanks

**Comment by petm5** [comment]
Linux on Chromebooks works with or without developer mode enabled. However, virtualization is required for it to work. You need to enable "Virtualization Technology" or "AMD-V" in the BIOS and then the option to install the Linux beta will show up.


Usually on chromebooks the developer mode switch is handled by the firmware. Escape+refresh+power and control+d won't work here though since Brunch only installs Chrome OS but doesn't replace the firmware. As such, running "crossystem dev-boot-signed-only=0" won't have any effect either.

If you installed Brunch with secure boot support, developer mode should be disabled by default. If you installed Brunch without secure boot or with the legacy bootloader, developer mode should be enabled by default.

You can tell if developer mode is enabled by typing "shell" in Crosh. If the command is not found, developer mode is disabled.

To enable/disable developer mode on Brunch you need to edit the kernel parameters through the GRUB2 configuration (this depends on how you installed Brunch, most likely the file is at /efi/boot/grub.cfg on partition 12) and add/remove "cros_debug" at the end of the line that starts with "linux".

**Comment by tejasraman** [comment]
Dev Mode is enabled by default in brunch.

**Comment by tejasraman** [comment]
Shell in crosh works on my HP by default :) - only dev mode allows that
just more proof

**Reply by minhnt812018** [reply]
hi, i need to install google search voice (google service) but after I installed and rebooted my PC, that service had been removed from OS, do you know how to bypass this issue? I don't want ChromeOS remove my google search voice after reboot or logout
